packages/flutter/lib/src/material/tabs.dart

```diff
@@ -419,7 +419,19 @@ class _IndicatorPainter extends CustomPainter {
   }
 
   static bool _tabOffsetsEqual(List<double> a, List<double> b) {
-    if (a?.length != b?.length)
+    // TODO(shihaohong): Null checks were added because of crashes in apps
```

Comment:
This TODO should just say that the following test should be replaced when a fix for https://github.com/flutter/flutter/issues/40014 is available.